packages/mui-base/src/SliderUnstyled/SliderUnstyled.tsx

```diff
@@ -281,7 +282,6 @@ const SliderUnstyled = React.forwardRef(function SliderUnstyled<
             })}
             style={{
               ...style,
-              pointerEvents: disableSwap && active !== index ? 'none' : undefined,
```

Comment:
Why is this logic not inside `getThumbProps()` of useSlider? It seems that we have to fix 3 different places at the same time.

Reply:
moved logic to `getThumbProps` but couldn't correctly type `thumbProps` hence had to use @ts-ignore.

https://github.com/mui/material-ui/pull/35998/files#diff-10ce33fc17ee7c4c1c1ec9123c97682153eef706ec3eaf1162827687bae42869R285